service/k8sclient/kubectl/kubectl.go

```diff
@@ -212,13 +211,13 @@ func (k *KubeCtl) Get(ctx context.Context, kind string, name string, res interfa
 }
 
 // Apply executes `kubectl apply` with given resource.
-func (k *KubeCtl) Apply(ctx context.Context, res meta.Object) error {
+func (k *KubeCtl) Apply(ctx context.Context, res interface{}) error {
```

Comment:
Oh, that's good. I tried to copy meta.Object and related types, and died on that hill.
But I'm afraid we would had to do it later…

Reply:
We can create our own interface if we will require it.